kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/steps/KroxyliciousSteps.java

```diff
@@ -31,6 +31,30 @@ public static void produceMessages(String namespace, String topicName, String bo
         KafkaUtils.produceMessageWithTestClients(namespace, topicName, bootstrap, message, numberOfMessages);
     }
 
+    /**
+     * Produce messages with kcat.
+     *
+     * @param namespace the namespace
+     * @param topicName the topic name
+     * @param bootstrap the bootstrap
+     * @param message the message
+     * @param numberOfMessages the number of messages
+     */
+    public static void produceMessagesWithKcat(String namespace, String topicName, String bootstrap, String message, int numberOfMessages) {
```

Comment:
As I mentioned else where I'm not sold on the idea of including the client type in the API (though I can see merits in doing so). Personally I think if we want to keep `produceMessagesWithKcat` it should be a connivance wrapper of something like: `produceMessages(String namespace, Job client, String topicName, String bootstrap, String message, int numberOfMessages)`

I'm not sure about using `Job` in the API or if we want a `KafkaClient` type (which is a job)